src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -391,6 +392,14 @@ void testParentQueryResult() throws Exception {
     assertParentQueryResult(new SnapshotBuilderService.ParentQueryResult(fieldValueList));
   }
 
+  @Test
+  void testRejectSnapshotAccessRequest() {
+    UUID id = UUID.randomUUID();
+    when(snapshotRequestDao.update(id, SnapshotAccessRequestStatus.REJECTED))
+        .thenReturn(new SnapshotAccessRequestResponse());
+    snapshotBuilderService.rejectSnapshotAccessRequest(id);
```

Comment:
There's no assert here; I would either assert the result of calling `rejectSnapshotAccessRequest(id)`, or instead of stubbing the `update` call, verify it. Either way would verify that the code implemented `reject` as expected.

so either
```
    var response = new SnapshotAccessRequestResponse());
    when(snapshotRequestDao.update(id, SnapshotAccessRequestStatus.REJECTED))
        .thenReturn(response);
    assertThat(snapshotBuilderService.rejectSnapshotAccessRequest(id), is(response));
```

or
```
    snapshotBuilderService.rejectSnapshotAccessRequest(id);
    verify(snapshotRequestDao.update(id, SnapshotAccessRequestStatus.REJECTED));
```